src/styles/base.css

```diff
@@ -210,9 +210,7 @@ strong {
 }
 
 a {
-  text-decoration: none;
   color: #fff;
```

Comment:
Should we have something else to identify links when they're not hovered? I may have misunderstood, but that's what I thought the original suggestion was about. The person was arguing that it's hard to find links on the page because they look like the rest of the text and you can only see them when you hover and they change color. Maybe we can keep our header and footer links like this, but make all other links (like in the body of reports) be a slightly different shade of orange when not hovered.

Reply:
Yup! I was removing `text-decoration: none`, so now they will all be underlined!

Reply:
Disregard that comment, I didn't see at first that links are underlined. 😄